test/STALK.js

```diff
@@ -24,9 +40,9 @@ describe('STALK', function(done) {
         'value': 5.305800926062761
       });
       delta.should.be.validSignalKDelta;
-      done();
```

Comment:
I don't think this can be correct. If the done() call is not triggered on output your test will pass always, even if the parser never emits anything.